cypress/support/pageBuilder/pbCreateCategoryAndBlocks.js

```diff
@@ -0,0 +1,102 @@
+import { GraphQLClient } from "graphql-request";
+
+function makeid(length) {
+    var result = "";
+    var characters = "abcdefghijklmnopqrstuvwxyz";
+    var charactersLength = characters.length;
+
+    for (var i = 0; i < length; i++) {
+        result += characters.charAt(Math.floor(Math.random() * charactersLength));
+    }
+    return result;
+}
+
+Cypress.Commands.add("pbCreateCategoryAndBlocks", (categoryVariables, numBlocks) => {
+    cy.login().then(user => {
+        const client = new GraphQLClient(Cypress.env("GRAPHQL_API_URL"), {
+            headers: {
+                authorization: `Bearer ${user.idToken.jwtToken}`
+            }
+        });
+
+        const createCategoryMutation = `
+      mutation CreateBlockCategory($data: PbBlockCategoryInput!) {
+        pageBuilder {
+          blockCategory: createBlockCategory(data: $data) {
+            data {
+              slug
+              name
+              icon
+              description
+              createdOn
+              createdBy {
+                id
+                displayName
+              }
+            }
+            error {
+              code
+              message
+              data
+            }
+          }
+        }
+      }
+    `;
+
+        const createBlockMutation = `
+      mutation CreatePageBlock($data: PbCreatePageBlockInput!) {
+        pageBuilder {
+          pageBlock: createPageBlock(data: $data) {
+            data {
+              id
+              blockCategory
+              preview
+              name
+              content
+              createdOn
+              createdBy {
+                id
+                displayName
+                type
+              }
+            }
+            error {
+              code
+              message
+              data
+            }
+          }
+        }
+      }
+    `;
+
+        const createCategoryPromise = client
+            .request(createCategoryMutation, { data: categoryVariables })
+            .then(response => response.pageBuilder.blockCategory.data);
+
+        return createCategoryPromise.then(categoryData => {
+            const categorySlug = categoryData.slug;
+
+            const createBlocksPromises = [];
+
+            for (let i = 0; i < numBlocks; i++) {
+                const blockVariables = {
+                    // Define your block variables here
```

Comment:
Redundant comment probably. 

I'd rather put a comment that explains why we're sending `content` and `preview` as `{}`. Basically we're doing it to satisfy the API, and for testing purposes, we can leave the as blank.

P.S. In the future, we might need to expand this command and allow for passing down `content` / `preview`. But yeah, we'll think about this when the need arises.